docs/ocaml/index.html

```diff
@@ -0,0 +1,2 @@
+<!DOCTYPE html>
+<html xmlns="http://www.w3.org/1999/xhtml"><head><title>index (bitwuzla.index)</title><link rel="stylesheet" href="../odoc.css"/><meta charset="utf-8"/><meta name="generator" content="odoc 1.5.2"/><meta name="viewport" content="width=device-width,initial-scale=1.0"/><script src="../highlight.pack.js"></script><script>hljs.initHighlightingOnLoad();</script></head><body><div class="content"><header><nav><a href="../index.html">Up</a> – bitwuzla</nav><h1 id="bitwuzla-index"><a href="#bitwuzla-index" class="anchor"></a>bitwuzla index</h1><nav class="toc"><ul><li><a href="#library-bitwuzla.c">Library bitwuzla.c</a></li><li><a href="#library-bitwuzla.z">Library bitwuzla.z</a></li></ul></nav></header><h2 id="library-bitwuzla.c"><a href="#library-bitwuzla.c" class="anchor"></a>Library bitwuzla.c</h2><p>The entry point of this library is the module: <a href="Bitwuzla_c/index.html"><code>Bitwuzla_c</code></a>.</p><h2 id="library-bitwuzla.z"><a href="#library-bitwuzla.z" class="anchor"></a>Library bitwuzla.z</h2><p>The entry point of this library is the module: <a href="Bitwuzla_z/index.html"><code>Bitwuzla_z</code></a>.</p></div></body></html>
```

Comment:
I would like to have more general documentation on this index page. Ideally, eventually, a quickstart guide and some examples. But at the very least some basic information about the two modules. Should this even be two modules? What is `Bitwuzla_z`?

Reply:
I am not sure that `odoc` allows a lot of customizations. I think the idea was to have a consistent "style" across OCaml documentations.

I will try to write some information/guide but I will not have a lot of free time this week so do not expect it before next week-end.

About `Bitwuzla_z`, it is an optional binding allowing to directly extract the value of bitvector to a `Zarith` integer (using `gmp` too). I may change it from optional to required and merge it with `Bitwuzla_c`.
Yet, the end goal is to provide a module `Bitwuzla.Session` that will be a type safe API build on top of `Bitwuzla_c`.

Reply:
Sounds good! No rush, whenever you have time.